jvb/src/main/kotlin/org/jitsi/videobridge/relay/Relay.kt

```diff
@@ -763,6 +764,26 @@ class Relay @JvmOverloads constructor(
         s?.expire()
     }
 
+    val incomingBitrateBps: Double
+        get() = transceiver.getTransceiverStats().rtpReceiverStats.packetStreamStats.getBitrateBps() +
+            synchronized(endpointsLock) {
+                relayedEndpoints.values.sumOf { it.getIncomingStats().getBitrateBps() }
+            }
+
+    val incomingPacketRate: Long
+        get() = transceiver.getTransceiverStats().rtpReceiverStats.packetStreamStats.packetRate +
+            synchronized(endpointsLock) {
+                relayedEndpoints.values.sumOf { it.getIncomingStats().packetRate }
+            }
+
+    val outgoingBitrateBps: Double
+        get() = transceiver.getTransceiverStats().rtpReceiverStats.packetStreamStats.getBitrateBps() +
```

Comment:
I think this should be `getTransceiverStats().outgoingPacketStreamStats`, same below for packet rate

Reply:
Yep, I noticed that just after the initial PR